src/sqlfluff/rules/L028.py

```diff
@@ -138,8 +143,21 @@ def _visit_queries(self, query: Query) -> Iterator[LintResult]:
                     self._fix_inconsistent_to,
                     fixable,
                 )
-        for child in query.children:
-            yield from self._visit_queries(child)
+        children = list(query.children)
+        # 'query.children' includes CTEs and "main" queries, but not queries in
+        # the "FROM" list. We want to visit those as well.
+        for a in select_info.table_aliases if select_info else []:
```

Comment:
```suggestion
        for a in (select_info.table_aliases if select_info else []):
```
perhaps brackets or a variable, the turnary  in line with a loop is an overload for me